Can't get probot/@octokit/rest/plugin-rest-endpoint-methods to compile

There seems to be an incompatibility between `@octokit/rest`, `probot`, and `plugin-rest-endpoint-methods`, such that I cannot get the modules to compile together:

```
node_modules/@octokit/rest/node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:1801:60 - error TS2339: Property 'POST /repos/{owner}/{repo}/forks{?org,organization}' does not exist on type 'Endpoints'.

1801             parameters: RequestParameters & Omit<Endpoints["POST /repos/{owner}/{repo}/forks{?org,organization}"]["parameters"], "baseUrl" | "headers" | "mediaType">;
                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@octokit/rest/node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:1802:33 - error TS2339: Property 'POST /repos/{owner}/{repo}/forks{?org,organization}' does not exist on type 'Endpoints'.

1802             response: Endpoints["POST /repos/{owner}/{repo}/forks{?org,organization}"]["response"];
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/probot/node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:1801:60 - error TS2339: Property 'POST /repos/{owner}/{repo}/forks{?org,organization}' does not exist on type 'Endpoints'.

1801             parameters: RequestParameters & Omit<Endpoints["POST /repos/{owner}/{repo}/forks{?org,organization}"]["parameters"], "baseUrl" | "headers" | "mediaType">;
                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/probot/node_modules/@octokit/plugin-rest-endpoint-methods/dist-types/generated/parameters-and-response-types.d.ts:1802:33 - error TS2339: Property 'POST /repos/{owner}/{repo}/forks{?org,organization}' does not exist on type 'Endpoints'.

1802             response: Endpoints["POST /repos/{owner}/{repo}/forks{?org,organization}"]["response"];
```

I can't seem to figure out an approach to make them play nice together.

Refs: https://github.com/googleapis/repo-automation-bots/pull/1660

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "@octokit/core": "^3.2.3",
     "@octokit/plugin-paginate-rest": "^2.6.2",
     "@octokit/plugin-request-log": "^1.0.2",
-    "@octokit/plugin-rest-endpoint-methods": "5.0.0"
+    "@octokit/plugin-rest-endpoint-methods": "5.0.1"
   },
   "devDependencies": {
     "@octokit/auth": "^3.0.1",

